crates/rust-analyzer/src/to_proto.rs

```diff
@@ -1123,7 +1123,7 @@ mod tests {
                 (
                     "arg",
                     Some(
-                        "fffffffe",
+                        "fffffffd",
```

Comment:
We want `&arg` to be sorted before `arg` in the completion list, because `&arg` is an exact type match here and `arg` is not. So they should not have the same value for `sort_text`.